changelogs/fragments/4839-fix-VirtualMediaInsert-Supermicro.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - redfish_command - Supermicro doesn't clear the ImageName after unmounting it so we now just check
+    if VirtualMedia.Instered == False
```

Comment:
This looks like something not understandable for **users** of this module. Can you reformulate it?

Also, please check out the changelog entry format in https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment, you need the PR URL in here.